packages/gatsby-theme-docs/README.md

```diff
@@ -1,3 +1,168 @@
-# Gatsby theme for commercetools docs
+# Core Gatsby theme
 
-TBD
+This is the core Gatsby theme for building commercetools documentation websites.
+
+## Getting started
+
+To create a new documentation website you need to install this theme and its peer dependencies:
+
+```
+npx install-peerdeps --dev @commercetools-docs/gatsby-theme-docs
+```
+
+### Choose a path prefix
+
+All commercetools documentation websites are served under `docs.commercetools.com`. In order to make this work, all documentation websites must be bundled for production using a `pathPrefix`. This value determines the URL path where the website is served from.
+
+For example, for the "Custom Applications" website, the path prefix is `/custom-applications`.
+
+The `pathPrefix` is configured in the `gatsby-config.js` file.
+
+### Folder structure
+
+The project structure should contain at least the following files and folders:
+
+```
+├── .eslintrc.yml
+├── gatsby-config.js
+├── package.json
+└── src
+    ├── content
+    │   ├── files
+    │   └── index.mdx
+    ├── data
+    │   └── navigation.yaml
+    └── images
+```
+
+- `.eslintrc.yaml`: in case you're using a monorepository, you need to provide this file with an empty object `{}`
+
+- `gatsby-config.js`: this is required for a Gatsby website and should contain the website specific configuration. At the very least, the core Gatsby theme must be listed in the plugins section. You are free to provide more plugins as you need for your specific website.
+
+  ```js
+  module.exports = {
+    pathPrefix: '/change-me',
+    siteMetadata: {
+      title: 'CHANGE_ME',
+      description: 'CHANGE_ME',
+    },
+    plugins: [
+      {
+        resolve: '@commercetools-docs/gatsby-theme-docs',
+        options: {
+          websiteKey: 'change-me',
+        },
+      },
+    ],
+  };
+  ```
+
+  Available options for the theme plugin are:
+
+  - `websiteKey` (**required**): the identifier of the website, without whitespaces. Usually this value would be the same as the `pathPrefix` without the leading slash.
+
+  - `gaTrackingId` (**required**): this is the Google Analytics tracking ID. The recommended ID to be used is: `UA-38285631-3`.
+
+    > For test websites the field should not be set.
+
+  - `createNodeSlug` (_optional_): in case you need to have more control over the creation of the page slugs, you can implement this function. This is useful if for example your website has content files in other file system locations and you want to provide a more meaningful URL path.
```

Comment:
I don't think this should be documented so prominently.  If you want a more meaningful path, provide more meaningful folder and file structures.   I'm really struggling with refactoring of the existing site because there's such an unpredictable mix of manual and conventional URLs.

Reply:
>  I don't think this should be documented so prominently.

How do you suggest to document it then?

Reply:
Since I don't see a need for the feature I would not document it at all for now.  But if you have a good use case example we can keep it and then document it here where you have it.

Reply:
It's already used in app-kit:

https://github.com/commercetools/merchant-center-application-kit/blob/bca85524b7a24329ca4c786265a90c224c9e2ad1/website/gatsby-config.js#L35-L46

That's why we have it and why It's documented 😅